lib/bittrex/order/order_status.ex

```diff
@@ -18,6 +18,8 @@ defmodule Bittrex.Order.OrderStatus do
     }
   end
 
+  def new(_), do: nil
```

Comment:
Could you document why this is not matching any of the cases so we keep track of what was the issue.

Maybe post the response so we could see why it wasn't matching

Reply:
`/market/buylimit` and `/market/selllimit` have the response type of:
``` Javascript
{
	"success" : BOOLEAN,
	"message" : STRING,
	"result" : {
	    "uuid" : STRING
	}
}
```

The `Opened` and `Closed` fields are missing therefore this match fails.